CHANGELOG.md

```diff
@@ -59,6 +59,10 @@ v0.24 + 1
 * `git_blob_create_fromchunks()` has been removed in favour of
   `git_blob_create_fromstream()`.
 
+* `clone_checkout_strategy` has been removed from
+  `git_submodule_update_option` as it is no longer necessary after
+  removing the `GIT_CHECKOUT_SAFE_CREATE` flag in favour of the 'GIT_CHECKOUT_SAFE' flag.
```

Comment:
I think that this could be worded in a way that suggests why it is no longer necessary or what the end-result is.  What do you think about:

>  `clone_checkout_strategy` has been removed from `git_submodule_update_option`.  The checkout strategy used to clone will be the same strategy specified in `checkout_opts`.